### DISCUSSION TITLE
My smtp server not sending mails

### DISCUSSION THREAD

**Discussion by jossybenson** [postalserver/postal#1662] [Help with using Postal] [answered]
I discovered today my postal smtp not sending mails, I host it with digitalocean server droplet, I when ahead to reset or rebuilt my droplet to format everything and reinstall postal again but still not working, and I only install postal on that server no any other program apart from postal on that server. I need urgent help please.

**Comment by willpower232** [comment, marked as answer]
https://github.com/postalserver/postal/discussions/1661
